app/src/main/kotlin/co/netguru/baby/monitor/client/common/YesNoDialog.kt

```diff
@@ -44,19 +45,22 @@ class YesNoDialog : androidx.fragment.app.DialogFragment() {
         private const val KEY_TITLE = "key:title"
         private const val KEY_MESSAGE = "key:message"
         private const val KEY_POSITIVE = "key:positive"
+        private const val KEY_NEGATIVE = "key:negative"
         private const val KEY_REQUEST_CODE = "key:request_code"
 
         fun newInstance(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/4b723c758e07b01bb18370eb08a478ace123b18b/app/src/main/kotlin/co/netguru/baby/monitor/client/common/YesNoDialog.kt#L51"></span>The function newInstance has too many parameters. The current threshold is set to 6.</td>
    </tr>
  </tbody>
</table>